### DISCUSSION TITLE
PO-TABLE - Desabilitar ações de uma única linha

### DISCUSSION THREAD

**Discussion by thiagogoncalves** [po-ui/po-angular#764] [Tire suas dúvidas]
Gostaria de colocar uma validação nas ações nas linhas habilitando ou desabilitando uma ação de acordo com o valor da linha. Na documentação eu vi que é possível desabilitar a opção para todas as linhas apenas. É possível desabilitar uma opção somente de uma linha da tabela ?

**Comment by jhonyeduardo** [comment]
Boa tarde @thiagogoncalves,

É possível sim, montei este ambiente para mostrar como: https://stackblitz.com/edit/po-ui-urubvu (stackblitz não carrega fonte corretamente, com isso os ícones não aparecem corretamente) 

Qualquer dúvida estou a disposição.
